packages/connectkit-next-siwe/src/configureSIWE.tsx

```diff
@@ -61,12 +80,16 @@ const getSession = async <TSessionData extends Object = {}>(
 const logoutRoute = async (
   req: NextApiRequest,
   res: NextApiResponse<void>,
-  sessionConfig: IronSessionOptions
+  sessionConfig: IronSessionOptions,
+  afterCallback: RouteHandlerOptions['afterLogout']
```

Comment:
@0xkwad i believe all these instances of `afterCallback` require to be optional since it will sometimes be set as `undefined`?
(e.g `afterCallback?: RouteHandlerOptions['afterLogout']`

Reply:
Good point 👍